compiler-rt/lib/sanitizer_common/tests/sanitizer_common_test.cpp

```diff
@@ -376,7 +376,25 @@ TEST(SanitizerCommon, RemoveANSIEscapeSequencesFromString) {
   }
 }
 
-TEST(SanitizerCommon, InternalScopedString) {
+TEST(SanitizerCommon, InternalScopedStringAppend) {
+  InternalScopedString str;
+  EXPECT_EQ(0U, str.length());
+  EXPECT_STREQ("", str.data());
+
```

Comment:
nit: can you add another null-append here (checking that null-append to an empty string is also sane):

```
  str.Append("");
  EXPECT_EQ(0U, str.length());
  EXPECT_STREQ("", str.data());
```